packages/tupaia-web/src/features/Map/utils/useMapOverlayData.ts

```diff
@@ -21,24 +17,25 @@ const useEntitiesByType = (
   entityCode?: EntityCode,
   entityType?: string | null,
 ) => {
+  const snakeCaseEntityType = getSnakeCase(entityType!);
   return useEntitiesWithLocation(
     projectCode,
     entityCode,
     {
       params: {
         // Don't include the root entity in the list of entities for displaying data as the
-        // data visuals are for children of the root entity
-        includeRootEntity: false,
+        // data visuals are for children of the root entity, unless it is a root entity, ie. a country
+        includeRootEntity: snakeCaseEntityType === 'country',
```

Comment:
Setting this to `false` for country level data was resulting in 0 results, which meant there was no map overlay data appearing on the screen